indico/migrations/versions/20171117_1035_40d4569835b8_add_include_description_to_reminders.py

```diff
@@ -0,0 +1,27 @@
+"""Add include_description to reminders
+
+Revision ID: 40d4569835b8
+Revises: 
+Create Date: 2017-11-17 10:35:12.378432
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '40d4569835b8'
+down_revision = None
```

Comment:
There are already some alembic revisions in 2.1, so `down_revision` needs to point to the latest revision before this one (also in the "Revises" line in the comment above)

Reply:
Probably safest to just delete it and regenerate it

Reply:
Done